src/extensions/default/brackets-show-whitespace/styles/whitespace-colors-css.tmpl

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
@flukeout do you know what a `.tmpl` file is with CSS?  I've never seen this before.